src/pages/AddCredentials/AddCredentials.js

```diff
@@ -216,7 +216,7 @@ const Issuers = () => {
 					handleContinue={handleContinue}
 					availableCredentialConfigurations={availableCredentialConfigurations}
 					popupTitle={`${t('pageAddCredentials.popup.title')} ${selectedIssuer?.selectedDisplay.name}`}
-					popupMessage={`${t('pageAddCredentials.popup.messagePart1')} ${selectedIssuer?.selectedDisplay.name}${t('pageAddCredentials.popup.messagePart2')}`}
+					popupMessage={`${t('pageAddCredentials.popup.messagePart1')} ${selectedIssuer?.selectedDisplay.name}${t('pageAddCredentials.popup.messagePart2IssuerSelect')}`}
```

Comment:
This should use interpolation rather than splitting the translation string:

```suggestion
					popupMessage={t('pageAddCredentials.popup.message', { issuerName: selectedIssuer?.selectedDisplay.name })}
```

(and then use `{{issuerName}}` as a placeholder in the translation string:)

```
	"pageAddCredentials": {
		"popup": {
			"message": "You have selected {{issuerName}}. This credential issuer can provide credentials of the following types. By selecting one type and clicking on 'Continue', you will be redirected to the issuer.",
```

Reply:
You're right! It might be a good idea to review the wwwallet frontend for similar improvements.

Reply:
Looks like there are 5 occurrences including this one. :slightly_smiling_face:

Reply:
It may be better to implement this change from the origin repository